website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Starburst (Trino) configurations"
+id: "trino-configs"
+---
+
+
+## Model-level configuration
+
+### Session properties
+
+In some specific cases, there may be needed tuning through the Trino session properties only for a specific dbt model. In such cases, using the [dbt hooks](https://docs.getdbt.com/reference/resource-configs/pre-hook-post-hook) may come to the rescue:
+
+```sql
+{{
+  config(
+    pre_hook="set session query_max_run_time='10m'"
```

Comment:
Since it's in the session, should this be a SQL header use case? https://docs.getdbt.com/reference/resource-configs/sql_header

Reply:
That looks wrong... hm I always thought we wanted things to happen in the session to be SQL headers but I might be wrong.